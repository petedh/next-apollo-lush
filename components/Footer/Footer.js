import styles from './Footer.module.css'
const Footer =() => {
  return(
    <footer className={styles.footer}>
      <div className={styles.waveHolder}>
        <svg viewBox="0 0 500 150" preserveAspectRatio="none" className={styles.wave}>
          <path d="M0.00,92.27 C216.83,192.92 304.30,8.39 500.00,109.03 L500.00,0.00 L0.00,0.00 Z" ></path>
        </svg>
      </div>
      <p>Copyright © 1995–2021 Lush Retail Ltd.</p>
    </footer>
  )
}
export default Footer