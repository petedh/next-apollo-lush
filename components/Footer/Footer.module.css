.footer {
  width: 100%;
  height: 250px;
  background-color: #000;
}

.waveHolder {
  height: 150px;
  overflow: hidden;
  background-color: #000;
}

.wave {
  height: 100%;
  width: 100%;
}

.wave path {
  stroke: none;
  fill: #fff;
}

.footer p {
  border-top: 1px solid rgb(51, 51, 51);
  padding: 2rem 2rem 5rem;
  text-align: right;
  width: 100%;
  color: #eaeaea;
}
