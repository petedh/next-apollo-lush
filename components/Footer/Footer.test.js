import Footer from './Footer'
import renderer from 'react-test-renderer'
import '@testing-library/jest-dom'

it('renders Footer unchanged', () => {
  const tree = renderer.create(<Footer />).toJSON()
  expect(tree).toMatchSnapshot()
})
