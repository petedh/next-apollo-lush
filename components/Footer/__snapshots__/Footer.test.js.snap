// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Footer unchanged 1`] = `
<footer
  className="footer"
>
  <div
    className="waveHolder"
  >
    <svg
      className="wave"
      preserveAspectRatio="none"
      viewBox="0 0 500 150"
    >
      <path
        d="M0.00,92.27 C216.83,192.92 304.30,8.39 500.00,109.03 L500.00,0.00 L0.00,0.00 Z"
      />
    </svg>
  </div>
  <p>
    Copyright © 1995–2021 Lush Retail Ltd.
  </p>
</footer>
`;
